### ISSUE TITLE
Step back a request using breakpoints

### ISSUE THREAD

**Issue by pr0letariat** [zaproxy/zaproxy#3886]
Would be useful to have a feature similar to Burp's Repeater to step back a request as well as forward.

**Comment by psiinon** [comment]
Interesting.
Can you explain a bit more about this - what does it allow you to do?
What does 'step back a request' actually mean in this context?
Although I used burp a long time ago I'm not that familiar with it.

**Comment by pr0letariat** [comment]
Sure, so basically you send a request to the Repeater, you send the request within Repeater but then you can step back to the previous request and view the different response.

Hope this makes sense of it:

![out](https://user-images.githubusercontent.com/14975253/30476054-b7771c72-9a00-11e7-9be5-719a527fdba7.gif)

**Comment by thc202** [comment]
This feature would apply to Manual Request Editor dialogue [1] not breakpoints? There's an add-on called Requester (available in the marketplace) with functionality similar to Repeater.

[1] https://github.com/zaproxy/zap-core-help/wiki/HelpUiDialogsMan_req

**Comment by pr0letariat** [comment]
I think you're right that it should be in the Manual Request Editor. I haven't seen any functionality to step back though.

**Comment by thc202** [comment]
Yeah, that's not yet implemented. I think we have already an issue for this functionality but not sure if it's still open.
